### PR TITLE
Add demos for docs

### PR DESCRIPTION
**Pull request by wzshiming**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: site/content/en/docs/examples/istio.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    memory: 256Gi
+    pods: 110
+  capacity:
+    cpu: 32
+    memory: 256Gi
+    pods: 110
+EOF
+```
+
+## Deploy istio
```
Comment (windsonsea) [inline site/content/en/docs/examples/istio.md:47]:
```suggestion
## Deploy Istio
```

File: site/content/en/docs/examples/linkerd.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+More information about Linkerd can be found at [linkerd quick start].
+
+## Set up cluster
+
+``` bash
+kwokctl create cluster --runtime kind
+```
+
+## Deploy linkerd
```
Comment (windsonsea) [inline site/content/en/docs/examples/linkerd.md:15]:
```suggestion
## Deploy Linkerd
```

File: site/content/en/docs/examples/argo.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Set up cluster
+
+``` bash
+wget https://github.com/kubernetes-sigs/kwok/raw/main/stages/pod-fast.yaml
+sed 's/Job/Workflow/g' pod-fast.yaml > workflow-fast.yaml
+kwokctl create cluster --runtime kind -c workflow-fast.yaml
+```
+
+## Deploy argo
```
Comment (windsonsea) [inline site/content/en/docs/examples/argo.md:17]:
```suggestion
## Deploy Argo
```

File: site/content/en/docs/examples/argo.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+``` bash
+kubectl create namespace argo
+kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.4.8/install.yaml
+kubectl patch deploy argo-server -n argo --type=json -p='[{"op":"add","path":"/spec/template/spec/nodeName","value":"kwok-kwok-control-plane"}]'
+kubectl patch deploy workflow-controller -n argo --type=json -p='[{"op":"add","path":"/spec/template/spec/nodeName","value":"kwok-kwok-control-plane"}]'
+```
+
+## Create node
```
Comment (windsonsea) [inline site/content/en/docs/examples/argo.md:26]:
```suggestion
## Create Node
```

File: site/content/en/docs/examples/istio.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+title: "With Istio"
+---
+
+# Istio
+
+More information about Istio can be found at [istio quick start].
```
Comment (windsonsea) [inline site/content/en/docs/examples/istio.md:7]:
```suggestion
More information about Istio can be found at [Istio quick start].
```

File: site/content/en/docs/examples/istio.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+## Create Pod and Inject Sidecar
+
+``` bash
+kubectl label namespace default istio-injection=enabled
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.18/samples/bookinfo/platform/kube/bookinfo.yaml
+```
+
+[istio quick start]: https://istio.io/latest/docs/setup/getting-started/
```
Comment (windsonsea) [inline site/content/en/docs/examples/istio.md:62]:
```suggestion
[Istio quick start]: https://istio.io/latest/docs/setup/getting-started/
```

File: site/content/en/docs/examples/linkerd.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+---
+title: "With Linkerd"
+---
+
+# Linkerd
+
+More information about Linkerd can be found at [linkerd quick start].
```
Comment (windsonsea) [inline site/content/en/docs/examples/linkerd.md:7]:
```suggestion
More information about Linkerd can be found at [Linkerd quick start].
```

File: site/content/en/docs/examples/linkerd.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+## Create Pod and Inject Sidecar
+
+``` bash
+kubectl apply -f https://run.linkerd.io/emojivoto.yml
+kubectl get -n emojivoto deploy -o yaml | \
+  linkerd inject - | \
+  kubectl apply -f -
+```
+
+[linkerd quick start]: https://linkerd.io/getting-started/
```
Comment (windsonsea) [inline site/content/en/docs/examples/linkerd.md:68]:
```suggestion
[Linkerd quick start]: https://linkerd.io/getting-started/
```

File: site/content/en/docs/examples/argo.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
+---
+title: "With Argo"
+---
+
+# Argo
+
+More information about Argo can be found at [argo quick start].
```
Comment (windsonsea) [inline site/content/en/docs/examples/argo.md:7]:
```suggestion
More information about Argo can be found at [Argo quick start].
```

File: site/content/en/docs/examples/argo.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+EOF
+```
+
+## Test Workflow
+
+``` bash
+argo submit -n argo --watch https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/hello-world.yaml
+```
+
+[argo quick start]: https://argoproj.github.io/argo-workflows/quick-start/
```
Comment (windsonsea) [inline site/content/en/docs/examples/argo.md:64]:
```suggestion
[Argo quick start]: https://argoproj.github.io/argo-workflows/quick-start/
```

**Review by windsonsea — COMMENTED** [review]
Pick nits about typo.

**Comment by windsonsea** [comment]
Text is good and easy to understand to me now.

**Comment by wzshiming** [comment]
> Text is good and easy to understand to me now.

I made a few more changes 😄

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by wzshiming** [comment]
Fix typos 😄

**Comment by windsonsea** [comment]
lol.
/lgtm

**Comment by windsonsea** [comment]
wow, the CI keeps running to complete after 3 hours passed

**Pull request merged**